raiden/tests/unit/test_tokennetwork.py

```diff
@@ -208,15 +209,18 @@ def test_channel_data_removed_after_unlock(
         block_number=closed_block_number,
         block_hash=closed_block_hash,
     )
+    channel_state_after_closed = channel_closed_iteration.new_state.channelidentifiers_to_channels[
+        channel_state.identifier
+    ]
 
     settle_block_number = closed_block_number + channel_state.settle_timeout + 1
     channel_settled_state_change = ContractReceiveChannelSettled(
         transaction_hash=factories.make_transaction_hash(),
         canonical_identifier=channel_state.canonical_identifier,
         block_number=settle_block_number,
         block_hash=factories.make_block_hash(),
-        our_onchain_locksroot=factories.make_32bytes(),
-        partner_onchain_locksroot=EMPTY_MERKLE_ROOT,
+        our_onchain_locksroot=merkleroot(channel_state_after_closed.our_state.merkletree),
```

Comment:
now we must use a valid merkleroot value, otherwise the channel cleaning does not work.